app/java/src/main/java/com/google/cloudclientapi/CensusController.java

```diff
@@ -16,28 +16,65 @@
 
 package com.google.cloudclientapi;
 
-
+import com.google.gson.Gson;
 import java.util.Arrays;
 import java.util.List;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 
 @Controller
 public class CensusController {
+
+  private static final String PROCESSED_DATA_BUCKET = System.getenv().get("PROCESSED_DATA_BUCKET");
   private static final Logger logger = LoggerFactory.getLogger(CensusController.class);
 
   @GetMapping("/")
-  public String main(Model model) {
-    logger.info("Received request (TODO: data)");
+  public String main(
```

Comment:
nit: I think this is a bad method name. Better to call it `default` or `index`. The `main` method is confusing.

Reply:
Sounds good. Just renamed from `main` to `index`.